lib/tasks/upgrade.rake

```diff
@@ -0,0 +1,17 @@
+namespace :upgrade do
+  desc "Run sidekiq with old queues so it can finish deferred jobs"
+  task :run_legacy_queues do
+    system "bundle exec sidekiq --daemon -q maintenance -q dispatch -q delete_account -q http -q http_service"\
+           " -q export -q photos -q socket_webfinger -q mail -q receive_local -q receive -q receive_salmon"\
```

Comment:
Oh, I guess so. I wonder if we force retry them all whether they'll end up on the new queues when they fail again?